frontend/public/components/resource-dropdown.tsx

```diff
@@ -102,11 +109,17 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
 
   const autocompleteFilter = (text, item) => {
     const { model } = item.props;
+    const language = getActiveLanguage();
```

Comment:
I'm debating if this is the right way to set the language. In other contexts like `Intl.NumberFormat`, you wouldn't want to initialize the locale this way since we want to show the numbers based on the user's locale regardless of whether we have translations for it or now. I guess this context is a little different, however.

My other concern is that this helper has a loop in it. The logic in this function is going to run a lot (once for every resource and for every keystroke), so adding this here could have an impact on the performance.

I would probably just not pass the language to `toLocaleLowerCase` here and let it initialize from the user's system locale or just go back to `toLowerCase`.

Reply:
I'm just going to go back to toLowerCase. toLocaleLowerCase says the output will be similar in most cases except for things like Turkish, which we don't currently support anyway.